libraries/dialogs/README.md

```diff
@@ -174,6 +176,8 @@ CustomDialog is a dialog that has a fixed header and its content can take a cust
 | Field | Type | Description | Default Value |
 | ------------- |-------------|-------------| ------------- |
 | `view` | View | The custom view that will create the body of the dialog. | null |
+| `horizontalPadding` | Float | Horizontal padding will be applied. | 0dp |
```

Comment:
Why default is different than the above?

Reply:
Main purpose of custom dialog is set view from client. So if we want to use custom dialog, we have to set horizontalPadding all the time. I think we should handle padding from custom view